### ISSUE TITLE
Webhook notifications defaulting to GET?

### ISSUE THREAD

**Issue by cmacrowther** [karimsa/patrol#24]
First off - just wanted to thank you for making this app - I've been using it to monitor a few of my servers. I've been trying to send POST webhooks to pushover.net and for some reason, Patrol is automatically switching them to GET and causing them to fail.

Here is an example of what I'm using in my patrol.config:
```
name: **NAME**
db: data.db

## Map consisting of services to display on your statuspage. Each service
## can have multiple checks.
## All check commands are simply run using the default shell (/bin/sh).
services:
  WEB:
    checks:
      - name: Latency
        type: metric
        unit: ms
        interval: 60s
        cmd: 'curl -fsSL -w "%{time_total}" -o /dev/null **SITE**'
      - name: Server Status
        interval: 60s
        cmd: 'curl -fsSL -m 10 -o /dev/null **SITE**'
    on_failure:
      - webhook:
          method: post
          url: https://api.pushover.net/1/messages.json
          headers:
            'Content-Type': 'application/json'
          body: 'body here'
```

And here is what I'm seeing in the logs: 

```
"WEB": {
      "Checks": [
        {
          "Name": "Latency",
          "Interval": 60000000000,
          "Timeout": 180000000000,
          "Cmd": "curl -fsSL -w \"%{time_total}\" -o /dev/null **SITE**",
          "Type": "metric",
          "MetricUnit": "ms"
        },
        {
          "Name": "Server Status",
          "Interval": 60000000000,
          "Timeout": 180000000000,
          "Cmd": "curl -fsSL -m 10 -o /dev/null **SITE**",
          "Type": "boolean",
          "MetricUnit": ""
        }
      ],
      "OnFailure": [
        {
          "Webhook": {
            "URL": {
              "Scheme": "https",
              "Opaque": "",
              "User": null,
              "Host": "api.pushover.net",
              "Path": "/1/messages.json",
              "RawPath": "",
              "ForceQuery": false,
              "RawQuery": "",
              "Fragment": "",
              "RawFragment": ""
            },
            "Method": "GET",
            "Headers": {
              "Content-Type": "application/json"
            },
            "Body": "body here"
          }
        }
      ],
      "OnRecovered": null,
      "OnSuccess": null
    }
```

Am I doing something wrong?

**Comment by karimsa** [comment]
Hi, there! Thanks for the kind words. My ears are always open if you've got feature requests or bug reports!

This is definitely a bug in patrol. I just pushed up a fix. Can you try the unstable release of patrol and let me know if that fixes it for you (`ghcr.io/karimsa/patrol:unstable`)?

**Comment by cmacrowther** [comment]
That did the trick! Thanks for the super quick response (and action). I'll have sharpen up my Go so I can start contributing in the future 🙂

**Comment by karimsa** [comment]
Look forward to your contributions!

🚀 The fix + notification support has been released in [`v1.1.0`](https://github.com/karimsa/patrol/runs/2168372068?check_suite_focus=true)!
